go-controller/pkg/clustermanager/secondary_network_unit_test.go

```diff
@@ -210,6 +210,93 @@ var _ = ginkgo.Describe("Cluster Controller Manager", func() {
 			})
 		})
 
+		ginkgo.When("Attaching to a localnet network", func() {
+			const subnets = "192.168.200.0/24,fd12:1234::0/64"
+
+			var (
+				fakeClient         *util.OVNClusterManagerClientset
+				netInfo            util.NetInfo
+				originalFeatConfig config.OVNKubernetesFeatureConfig
+			)
+
+			ginkgo.BeforeEach(func() {
+				fakeClient = &util.OVNClusterManagerClientset{
+					KubeClient:            fake.NewSimpleClientset(&v1.NodeList{Items: nodes()}),
+					NetworkAttchDefClient: fakenadclient.NewSimpleClientset(),
+				}
+
+				originalFeatConfig = config.OVNKubernetesFeature
```

Comment:
maybe using 
Expect(config.PrepareTestConfig()).To(Succeed()) // reset defaults

is standard right?